File: SoloLearn_CodeCoach/Easy/CheerCreator/CheerCreator.cpp
#include <iostream>
using namespace std;

int main() {
    const string A = "High Five";
    const string B = "Ra!";
    const string C = "shh";

    int yards;
    cin >> yards;

    if (yards > 10) {
        cout << A << endl;
    } else if (yards < 1) {
        cout << C << endl;
    } else {
        for (int i = 0; i < yards; i++) {
            cout << B;
        }
        cout << endl;
    }
    return 0;
}

File: SoloLearn_CodeCoach/Easy/CheerCreator/CheerCreator.java
import java.util.Scanner;

public class CheerCreator {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int yards = input.nextInt();

        if (yards > 10) {
            System.out.println("High Five");
        } else if (yards < 1) {
            System.out.println("shh");
        } else {
            String to_print = "";

            for (int i = 0; i < yards; i++) {
                to_print += "Ra!";
            }

            System.out.println(to_print);

        }
    }
}
